x-pack/test_serverless/api_integration/test_suites/common/index_management/settings.ts

```diff
@@ -28,6 +28,8 @@ export default function ({ getService }: FtrProviderContext) {
     });
 
     it('should fetch an index settings', async () => {
+      this.onlyEsVersion('>=9');
```

Comment:
I'd rather that this test run on both 8 and 9. The test results say `Error: Expected setting "frozen" not found.` which leads me to believe the test is run from the 8.x branch against 9.0 - `main` doesn't check for `frozen`. I think we need two versions of this test on 8.x - one that runs against 8.x and one that runs against 9.0

Reply:
Yeah that sounds like a good idea, I created another PR for doing so https://github.com/elastic/kibana/pull/211219